lib/src/data/chat/chat_message.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:misskey_dart/misskey_dart.dart';
+import 'package:misskey_dart/src/converters/date_time_converter.dart';
+
+part 'chat_message.freezed.dart';
+part 'chat_message.g.dart';
+
+@freezed
+class ChatMessage with _$ChatMessage {
+  const factory ChatMessage({
+    required String id,
+    @DateTimeConverter() required DateTime createdAt,
+    String? text,
+    required String fromUserId,
+    User? fromUser,
+    String? toUserId,
+    User? toUser,
+    String? toRoomId,
+    ChatRoom? toRoom,
+    required List<ChatMessageReaction> reactions,
+  }) = _ChatMessage;
+
+  factory ChatMessage.fromJson(Map<String, dynamic> json) =>
+      _$ChatMessageFromJson(json);
+}
+
+@freezed
+class ChatMessageReaction with _$ChatMessageReaction {
+  const factory ChatMessageReaction({
+    required String reaction,
+    required User user,
```

Comment:
```suggestion
    UserLite? user,
```

ユーザーとのチャットでは常にnullになるようです

ref: https://github.com/misskey-dev/misskey/blob/2025.4.0-beta.0/packages/backend/src/models/json-schema/chat-message.ts#L73-L78
ref: https://github.com/misskey-dev/misskey/blob/2025.4.0-beta.0/packages/backend/src/core/entities/ChatEntityService.ts#L136-L137